src/Export/Field/CustomFieldTrait.php

```diff
@@ -0,0 +1,111 @@
+        $defaultLocale = $this->findLanguage(Defaults::LANGUAGE_SYSTEM)->getLocale()->getCode();
+
+        /*
+         * @param string $key
+         * @param string|array $storedValue
+         *
+         * @return array
+         */
+        return function (string $key, $storedValue) use ($usedLocale, $defaultLocale) {
+            try {
+                $customField = $this->getCustomField($key);
+            } catch (\InvalidArgumentException $e) {
+                return [$key => $storedValue];
+            }
+
+            if ($customField->getType() === CustomFieldTypes::SELECT) {
+                $options = array_filter($customField->getConfig()['options'], function (array $option) use ($storedValue) {
+                    return is_array($storedValue) ? in_array($option['value'], $storedValue) : $option['value'] === $storedValue;
+                });
+
+                $translatedOptionValue = implode('#', array_map(function (array $option) use ($usedLocale, $defaultLocale): string {
+                    return array_key_exists('label', $option) && count($option['label']) > 0
+                        ? $option['label'][$usedLocale] ?? $option['label'][$defaultLocale]
+                        : $option['value'];
+                }, $options));
+            }
+
+            $label = $customField->getConfig() !== null && array_key_exists('label', $customField->getConfig())
+                ? ($customField->getConfig()['label'][$usedLocale] ?? $customField->getConfig()['label'][$defaultLocale])
+                : $key;
+
+            return [$label => $translatedOptionValue ?? $storedValue];
+        };
+    }
+
+    private function getFields(Entity $entity): array
+    {
+        $customFields = $entity->getTranslation('customFields') ?? [];
+
+        if (!empty($customFields)) {
+            if (!empty($this->exportSettings->getDisabledCustomFields())) {
+                $excludedCustomFields  = $this->customFieldsService->getCustomFieldNames($this->exportSettings->getDisabledCustomFields());
+                $customFields          = array_diff_key($customFields, array_flip($excludedCustomFields));
+            }
+        }
+
+        if ($entity instanceof Category) {
+            unset($customFields[OmikronFactFinder::CMS_EXPORT_INCLUDE_CUSTOM_FIELD_NAME]);
+        }
+
+        return $customFields;
+    }
+
+    private function getCustomField(string $key): CustomFieldEntity
+    {
+        if (!isset($this->loadedFields[$key])) {
+            $customField = $this->customFieldRepository->search(
+                (new Criteria())->addFilter(new EqualsFilter('name', $key)),
+                new Context(new SystemSource())
+            )->first();
+            if (!$customField) {
+                throw new InvalidArgumentException('There is no custom field with a given key');
```

Comment:
```suggestion
                throw new \InvalidArgumentException(sprintf('There is no custom field with a given key %s', $key));
```

it throws me an error as InvalidArgumentException is not imported in `use` statements
![image](https://user-images.githubusercontent.com/4511883/129140519-d25d5218-165b-4758-a247-ce0981a89782.png)